code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -50,12 +50,9 @@
 	. = ..()
 	for(var/I = 1 to length(contents))
 		var/obj/item/reagent_containers/food/snacks/donut/donut = contents[I]
-		var/icon/new_donut_icon = icon('icons/obj/food/containers.dmi', "donut_[donut.donut_sprite_type]")
-		new_donut_icon.Shift(EAST, 3 * (I-1))
+		var/icon/new_donut_icon = icon('icons/obj/food/containers.dmi', "[(I - 1)]donut[donut.donut_sprite_type]")
```

Comment:
do we really need 6 different donut sprites for each donut type, when they are just translated right? can't you get the shift system to work with this?

Reply:
The old system translated, but the new one unfortunately doesn't work that way with how the donuts are shaded